### ISSUE TITLE
Alternatives? Update to be safe again?

### ISSUE THREAD

**Issue by GitHubber78** [iam4x/pokemongo-webspoof#452]
Hi!

Any chance to get an update here which makes it safe to use again?

Or do you know safe alternatives to this method here? Preferably for iOS, But if there is no safe alternative for iOS, what is the suggestion for Android?

Regards,

GH

**Comment by guerrini** [comment]
For traceability, GitHubber78 question a follow-on to #451.
Could users have raised suspicion with something as simple as playing a location in a different time zone, but the phone time still set based on current physical time zone?  Seems it would be easy for PoGo to check phone time against location and flag account if different.

**Comment by maxfremars** [comment]
This tool is still save. 
If you fake the GPS via xCode, Pokemon Go can't figure this out. It "thinks" the GPS is valid.

The biggest problem here is :

You live in e.g. New York and you fake the GPS to be in L.A. You stop xCode and 1h later you open Pokemon Go in New York again. Niantic detects a huge location jump within an hour and Niantic thinks, you faked the gps and bans you.

**Comment by trisapple** [comment]
Sorry but I couldn’t think of any other alternatives. 

It’s still safe to use, but just don’t fully spoof your account. 

I got my baby account permanently terminated, but not my main account by using this app. I fully spoof my baby account, and partially spoof my main account. 

This may seem complicated, but what I think is that an account cannot have their travel type as “teleport” speed exceed a certain percentage of all their travel in the game.

**Comment by GitHubber78** [comment]
When I received my ban, I was only spoofing in my little home town. And I was not jumping from location to location (just 3 miles from one spot to another), I was "walking" with the spoof.
And I also played the same account "legit" from another iPhone, without the spoof installed.
So I don't think it was due to "false" spoofing.

**Comment by mactechh** [comment]
You played the same account, without the spoof on one device, and a second time with the spoof installed on another phone? So the same account was at different places at he same time? No wonder you got banned.

**Comment by maxfremars** [comment]
It looks like, the bans don’t have something to do with the tool.

Maybe the tool is still save to use, when used correctly.

**Comment by GitHubber78** [comment]
No, I did not play the same account on different locations on the same time.

E.g. in the morning legit on my iPhone in my hometown, later on in the evening from spoofing on my iPad.

And that for almost 2 years without any issues.

But then, all out of a sudden, the ban...

**Comment by Snoverfield** [comment]
I always play with "bike speed" and auto travel routes unless no routes available. I follow "cool down" rules. I don't jump around a lot. I usually after not being in game for over 2 hours will port to a location and take advantage of nests and look for shinies. I was fine for a really long time. But my baby alt account last month was basically perma banned in like 5 days after first warning. And now got my first warning on my main account last week. I play very safe. Not sure why I'm getting detected now. Afraid to keep using this. Tell me why I shouldn't. :/

**Comment by pxhll** [comment]
Is on of your account a trainer club and the other a google login? I observed that problems are always arising when switching between those two login mtheods.

**Comment by Snoverfield** [comment]
Actually yes. My small account was a trainer club and my main is google. I was fine for the longest time but maybe a new update screwed me over. Yet only my little was getting hit with bans and my main was hit like 3 weeks later and at the time wasn’t using my little account cause of it being banned and all. But I’ve heard just switching accounts a lot is bad. I’ve been meaning to get a handy down second phone to run my alt. I’ll just have to do that still. But still a little worried running my main.
